Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -819,23 +820,32 @@ def exception(message):
 
 
 async def slack_loop():
+    exception_await_seconds = 1
     while True:
-        # SocketModeClient does not respect environment variables for ssl verification.
-        # Instead we use a custom session.
-        session = aiohttp.ClientSession(connector=aiohttp.TCPConnector(verify_ssl=VERIFY_CERT))
-        slack_logger = SlackLogger()
-        client = SocketModeClient(
-            app_token=APP_TOKEN,
-            web_client=ASYNC_CLIENT,
-            logger=slack_logger  # type: ignore
-        )
-        client.aiohttp_client_session = session
-        client.socket_mode_request_listeners.append(listen)  # type: ignore
         try:
-            await client.connect()
-            await asyncio.sleep(float("inf"))
+            slack_logger = SlackLogger()
+            client = SocketModeClient(
+                app_token=APP_TOKEN,
+                web_client=ASYNC_CLIENT,
+                logger=slack_logger  # type: ignore
+            )
+            client.socket_mode_request_listeners.append(listen)  # type: ignore
+
+            try:
+                await client.connect()
+                await asyncio.sleep(float("inf"))
+            except Exception as e:
+                await handle_listen_error(f"An error occurred {str(e)}")
+            finally:
+                try:
+                    await client.disconnect()
+                except Exception as e:
+                    demisto.debug(f"Failed to close client. - {e}")
+            exception_await_seconds = 1
         except Exception as e:
-            await handle_listen_error(f"An error occurred {str(e)}")
+            demisto.debug(f"Exception in long running loop, waiting {exception_await_seconds} - {e}")
+            await asyncio.sleep(exception_await_seconds)
+            exception_await_seconds *= exception_await_seconds
```

Comment:
This is the fix for the ghost. I'd rather get this out sooner than later due to it's impact to customers.

Reply:
This doesn't play nicely with the other changes. I'll need to re-work what we do here. Disregard for now.